src/sidebar/services/router.js

```diff
@@ -61,13 +61,31 @@ export default function router($window, store) {
     return url;
   }
 
+  let didRegisterPopstateListener = false;
+
   /**
    * Synchronize the route name and parameters in the store with the current
    * URL.
+   *
+   * The first call to this method also registers a listener for future back/forwards
+   * navigation in the browser.
    */
   function sync() {
     const { route, params } = currentRoute();
     store.changeRoute(route, params);
+
+    // Setup listener for back/forward navigation. We do this in `sync()` to avoid
```

Comment:
Extreme pedantry alert: _Setup_ is the noun form; I think you want _set up_ here

Reply:
Pedantry appreciated :)